pkgs/by-name/pi/picom-pijulius/package.nix

```
@@ -2,17 +2,18 @@
   picom,
   lib,
   fetchFromGitHub,
-  pcre
+  pcre,
+  nix-update-script
 }:
 picom.overrideAttrs (previousAttrs: {
   pname = "picom-pijulius";
-  version = "8.2-unstable-2024-04-30";
+  version = "Next-unstable-2024-06-06";
```

The versioning scheme is a bit add. It should be like this per https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

```suggestion
  version = "8.2-unstable-2024-04-30";
```

---

This comes directly from nix-update, run with the script in maintainers/scripts/update.nix

---

You can use `unstableGitUpdater` and run it again! That'll give you the correct version formatting.